NitroxPatcher/Patches/Stalker_CheckLoseTooth_Patch.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Harmony;
+using NitroxModel.Helper;
+using System;
+using System.Reflection;
+using UnityEngine;
+
+namespace NitroxPatcher.Patches
+{
+    public class Stalker_CheckLoseTooth_Patch : NitroxPatch
+    {
+        public static readonly Type TARGET_CLASS = typeof(Stalker);
+        public static readonly MethodInfo TARGET_METHOD = TARGET_CLASS.GetMethod("CheckLoseTooth", BindingFlags.NonPublic | BindingFlags.Instance);
+
+        public static bool Prefix(Stalker __instance, GameObject target)
+        {
+            float hardness = 0f;
```

Comment:
Maybe we skip the naming `hardness` and just say probability or something similar.

Reply:
Should I merge the two probabilities (50% and h%) or given my explanations above it makes semantic sense to keep them separate? I like to also keep the reference to the properties of the material semantically for future reference. If something is hard, it should make it easier for teeth to drop. If we call it "probability" we remove this semantic reference. Do my arguments convince you? I'm fine either way.